webservices/legal_docs/current_cases.py

```diff
@@ -362,10 +401,10 @@ def get_sorted_respondents(participants):
     """
     Returns the respondents in a MUR sorted in the order of most important to least important
     """
-    SORTED_RESPONDENT_ROLES = ['Primary Respondent', 'Respondent', 'Previous Respondent']
+    SORTED_RESPONDENT_ROLES = ["Primary Respondent", "Responden", "Previous Respondent"]
```

Comment:
Is this missing a `t`?

` SORTED_RESPONDENT_ROLES = ["Primary Respondent", "Responden", "Previous Respondent"]`

Reply:
good catch. Will fix the spelling